pyxem/signals/insitu_diffraction2d.py

```diff
@@ -89,26 +88,24 @@ def get_time_series(self, roi=None, time_axis=2):
     def get_drift_vectors(
         self, time_axis=2, reference="cascade", sub_pixel_factor=10, **kwargs
     ):
-        """
-        Calculate real space drift vectors from time series of images
+        """Calculate real space drift vectors from time series of images
 
-         Parameters
+        Parameters
         ----------
-        s: Signal2D
+        s: :class:`~hyperspy.api.signals.Signal2D`
             Time series of reconstructed images
         reference: 'current', 'cascade', or 'stat'
-            reference argument passed to hs.signals.Signal2D.estimate_shift2D()
+            reference argument passed to :meth:`~hyperspy.api.signals.Signal2D.estimate_shift2D`
             function. Default is 'cascade'
         sub_pixel_factor: float
-            sub_pixel_factor passed to hs.signals.Signal2D.estimate_shift2D()
+            sub_pixel_factor passed to :meth:`~hyperspy.api.signals.Signal2D.estimate_shift2D`
             function. Default is 10
-        **kwargs:
-            Passed to the :meth:`get_time_series` function
+        kwargs:
```

Comment:
I find **kwargs renders better in the docs than just kwargs. You can drop the colon here.